hardening/kickstart/main.fmf

```diff
@@ -27,9 +27,7 @@ adjust:
     because: we want to run virtualization on x86_64 only
   - when: distro < rhel-10
     enabled: false
-    because: >
-        too late in the RHEL-8 lifecycle for this,
-        TODO enable this on RHEL-9 once OpenSCAP v1.4 is there
+    because: on RHEL <= 9 oscap-anaconda-addon is used instead
```

Comment:
Sorry to nitpick, but I think I would prefer a more concrete reason, ie. `because: upstream content kickstarts are not available for RHEL >= 10` or something verifiable like that.